tests/Dictionary_1.py

```diff
@@ -124,14 +124,14 @@ def testBadDictionary(self):
         p = Policy(self.getTestDictionary("conflict_occurs_policy_1.paf"))
         ve = ValidationError("Dictionary_1.py", 1, "testBadDictionary")
         d.validate(p, ve.cpp)
-        self.assert_(ve.getErrors("1to0") == ValidationError.TOO_MANY_VALUES)
-        self.assert_(ve.getErrors("2to1") == ValidationError.NOT_AN_ARRAY)
+        self.assert_(ve.getErrors("1to0") == int(ValidationError.TOO_MANY_VALUES))
+        self.assert_(ve.getErrors("2to1") == int(ValidationError.NOT_AN_ARRAY))
```

Comment:
This is needed because `getErrors` does not return an enum type, instead it returns `int` because multiple types of errors can be combined  in the return value (with bitwise or).
Thus `int` is needed (unfortunately) because pybind11 wrapped enums don't compare equal to their underlying integer type. I'll have a quick look if we can add a fix for this in upstream and will remove this if we can. 
Note that for most normal cases this wouldn't be a problem since enums would be compared to enums, which just works.